### PR TITLE
Add a page about XSL:FO rendering

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

**Comment by joewiz** [comment]
@adamretter Pages are automatically discovered via their `<keyword>` element, but they need to be manually entered into the documentation TOC page: https://github.com/eXist-db/documentation/blob/master/src/main/xar-resources/data/documentation/documentation.xml.

File: src/main/xar-resources/data/fo-render/fo-render.xml [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+
+  <sect1 xml:id="render-module">
+    <title>The <code>xslfo</code> module</title>
+
+    <para>eXist-db XSL:FO module allows you to do FO rendering from XQuery code using an appropriate
+      FO Renderer; eXist-db supports <link xlink:href="https://xmlgraphics.apache.org/fop/">Apache FOP</link> and <link xlink:href="http://www.renderx.com/tools/xep.html">RenderX XEP</link>. Previously there was also support
+      for AntennaHouse, but that was removed as it was unmaintained, we would welcome a new
+      contribution to support AntennaHouse again.</para>
+    <para>Selection of the renderer is configured in eXist-db's <code>conf.xml</code> file.
+      By default eXist-db ships with and is configured to use Apache FOP, see: <link linkend="renderx">Using
```
Comment (joewiz) [inline src/main/xar-resources/data/fo-render/fo-render.xml:28]: The FOP `@linkend` should point to `apachefop`, not `renderx`.

File: src/main/xar-resources/data/fo-render/fo-render.xml [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+    <para>Its function namespace is <literal>http://exist-db.org/xquery/xslfo</literal>.</para>
+
+    <!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
+
+    <sect2 xml:id="render">
+      <title>
+        <code>xsl:render()</code>
+      </title>
+
+      <para>The <code>xsl:render</code> function renders an FO document and returns you the result.
```
Comment (joewiz) [inline src/main/xar-resources/data/fo-render/fo-render.xml:41]: The namespace prefix for the `render()` function should be `xslfo`, not `xsl`.

**Review by joewiz — APPROVED** [review]
Thank you! LGTM - just a few minor issues, as noted inline.

**Comment by adamretter** [comment]
@joewiz Thanks for the speedy review, I just pushed an additional commit to address your suggestions :-)

**Review by dizzzz — APPROVED** [review]

**Comment by joewiz** [comment]
We have a CI error, which I don't think is related to this PR:

> Failed to execute goal com.github.eirslett:frontend-maven-plugin:1.12.0:gulp (gulp build) on project exist-documentation: Failed to run task: 'gulp.js ' failed. org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1) -> [Help 1]

... but the master branch is passing CI, so why would this PR suddenly throw an error?

**Comment by joewiz** [comment]
I see there's a dependabot PR for a newer version of eirslett:frontend-maven-plugin, so I'll work through the dependabot backlog and merge that PR. Hopefully that'll fix this.

**Comment by joewiz** [comment]
@dependabot rebase

**Comment by joewiz** [comment]
@dependabot why no love? 

ok, since master is green after applying the other dependabot PRs, I'll just merge and hope that master remains green.

**Pull request merged**